python/tests/tensor.py

```diff
@@ -1,45 +1,37 @@
 import numpy as np
 import pytest
 from numpy.testing import assert_equal
-from utils import large_tensor_map, tensor_map
+from .utils import large_tensor_map, tensor_map
 
 import equistore
 
 
 class TestTensorMap:
-    @pytest.fixture
-    def tensor(self):
-        return tensor_map()
+    def test_copy(self, tensor_map):
+        copy = tensor_map.copy()
+        block_1_values_id = id(tensor_map.block(0).values)
 
-    @pytest.fixture
-    def large_tensor(self):
-        return large_tensor_map()
-
-    def test_copy(self, tensor):
-        copy = tensor.copy()
-        block_1_values_id = id(tensor.block(0).values)
-
-        del tensor
+        del tensor_map
```

Comment:
this might not be doing what we want with `tensor_map` as a fixture. The goal is to remove all references to tensor_map, but pytest will still be holding one here.

Reply:
ahh you might be right. Can we test this somehow if all references are gone. Maybe pytest is clever :-)

Reply:
According to the pytest docs a fixture has a scope which by default is function. So every function should get its own independent copy.

https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-fixture

So it should be fine.

Reply:
The `tensors` is not doing copies right? It is creating a new `TensorMap` from scratch. `pytest` will give an independent instance to each test.

Reply:
I'll add a commit checking for this

Reply:
yep, pytest was keeping more references (6 total), I changed this test not to use fixture